packages/botonic-nlp/src/tasks/text-classification/process/prediction-processor.ts

```diff
@@ -9,11 +9,11 @@ export class PredictionProcessor {
 
   process(prediction: OutputData): Intent[] {
     const confidences = prediction.arraySync()[0]
-    const intents = this.generateIntents(confidences)
+    const intents = this.computeIntents(confidences)
```

Comment:
compute means there's a calculation/algorithm. I don't see it here. why not createIntents?